### PR TITLE
Add data name

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: apps/andi/assets/css/app.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@ body {
 
 .error-msg {
   color: red;
-  display: block;
+  display: block !important;
```
Comment (bbrewer-forge) [inline apps/andi/assets/css/app.scss:51]: wait, what?! is there a reason we have to do this for css we control? Or is this overriding a 3rd party component?

File: apps/andi/lib/andi/input_schemas/datasets.ex [hunk 2]
```diff
@@ -135,13 +135,16 @@ defmodule Andi.InputSchemas.Datasets do
   end
 
   def is_unique?(id, data_name, org_name) do
+    org_name = String.downcase(org_name)
+    data_name = String.downcase(data_name)
+
     get_all()
```
Comment (bbrewer-forge) [inline apps/andi/lib/andi/input_schemas/datasets.ex:141]: I'd love to see this done in a query, as it's likely more performant, but it's not a show stopped

File: apps/andi/assets/css/app.scss (hunk 1 above)
Comment (dunson062786) [inline apps/andi/assets/css/app.scss:51]: Why are you using !important?

File: apps/andi/lib/andi/input_schemas/datasets.ex (hunk 2 above)
Comment (bbrewer-forge) [reply apps/andi/lib/andi/input_schemas/datasets.ex:141]: Additionally, is there a test for this? That might make it slightly less painful to migrate this to a query later.

File: apps/andi/assets/css/metadata_form.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 .metadata-form-edit-section {
   grid-template-areas:
     "title last-updated"
+    "data-name ."
```
Comment (JarredOlson) [inline apps/andi/assets/css/metadata_form.scss:4]: do you want that space dot on the end?

File: apps/andi/assets/css/app.scss (hunk 1 above)
Comment (cameronmarsh) [reply apps/andi/assets/css/app.scss:51]: yeah we kind of just gave up going through the cascade but we will change this back

Comment (cameronmarsh) [reply apps/andi/assets/css/app.scss:51]: it was more just a temporary fix but we will change this back

File: apps/andi/lib/andi/input_schemas/datasets.ex (hunk 2 above)
Comment (cameronmarsh) [reply apps/andi/lib/andi/input_schemas/datasets.ex:141]: good point, there are some tests so we can do this

File: apps/andi/assets/css/metadata_form.scss (hunk 3 above)
Comment (bbrewer-forge) [reply apps/andi/assets/css/metadata_form.scss:4]: yeah, it's the goofy grid css way of adding a blank (or stretching it?) cell

Comment (cameronmarsh) [reply apps/andi/assets/css/metadata_form.scss:4]: yeah I think that is how you denote an empty space when defining a grid elsewhere in andi, but the styling is not currently lining up so I will look into it some more

**Pull request merged**